pandatools/PBookCore.py

```
@@ -110,7 +110,7 @@ def __init__(self, verbose=False):
         PsubUtils.check_proxy(self.verbose, None)
         # user name
         self.username = 'DEFAULT_USER'
-        username_from_proxy = MiscUtils.extract_voms_proxy_username()
+        username_from_proxy = PsubUtils.extract_voms_proxy_username()
         if username_from_proxy:
```

It would be better to fail rather than continuing with 'DEFAULT_USER' when username_from_proxy is None.

---

Done